tests/test_reduction_ops.py

```diff
@@ -250,6 +251,22 @@ def test_accuracy_cumsum(shape, dtype):
     gems_assert_close(res_out, ref_out, dtype, reduce_dim=shape[dim])
 
 
+@pytest.mark.parametrize("shape", REDUCTION_SHAPES)
+@pytest.mark.parametrize("dtype", FLOAT_DTYPES + INT_DTYPES + [torch.bool])
+def test_accuracy_nonzero(shape, dtype):
+    if dtype == torch.bool:
+        inp = torch.randint(0, 2, shape, dtype=torch.int, device="cuda").to(torch.bool)
+    else:
+        inp = torch.randint(0, 2, shape, dtype=dtype, device="cuda")
```

Comment:
The tested floating point values are seriously underrepresented. Better replace randint with rand.

Reply:
You might need to add 1d shapes to improve test coverage.

Reply:
> The tested floating point values are seriously underrepresented. Better replace randint with rand.

done, use randn for float cases

Reply:
> You might need to add 1d shapes to improve test coverage.

done, and 1D and MNK shapes for both cumsum and nonzero

Reply:
Thanks.